src/test/testScripts/diagnostics/testDiagnoticsAnsibleLocal.test.ts

```diff
@@ -0,0 +1,105 @@
+import * as vscode from 'vscode';
+import {
+  getDocUri,
+  activate,
+  testDiagnostics,
+  sleep,
+  updateSettings,
+  resetDefaultSettings,
+} from '../../helper';
+
+export function testDiagnosticsAnsibleLocal(): void {
+  describe('TEST FOR ANSIBLE DIAGNOSTICS', () => {
+    const docUri1 = getDocUri('diagnostics/ansible/1.yml');
+    const docUri2 = getDocUri('diagnostics/ansible/2.yml');
+
+    beforeEach(async () => {
+      await resetDefaultSettings();
+      await vscode.commands.executeCommand('workbench.action.closeAllEditors');
+    });
+
+    afterEach(async () => {
+      await resetDefaultSettings();
+    });
+
+    describe('Diagnostic test with ansible-lint', () => {
+      beforeEach(async () => {
+        await updateSettings('ansibleLint.enabled', true);
+      });
+
+      it('should complain about no task names', async () => {
+        await activate(docUri1);
+        await vscode.commands.executeCommand('workbench.action.files.save');
+
+        await sleep(2000); // Wait for the diagnostics to compute on this file
```

Comment:
Is there a more reliable way to wait?

Reply:
As of now, I could not find a better way to do it. Maybe if we find it in future, the changes can be made in the subsequent PRs.